src/nc_util.c

```diff
@@ -22,6 +22,9 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <netdb.h>
+#ifdef __linux
+#include <linux/version.h>
```

Comment:
The `__linux` specific code can be removed if `#ifdef SO_REUSEPORT` is used instead.